### ISSUE TITLE
Electron 4 crashes on start up on El Capitan

### ISSUE THREAD

**Issue by torjusb** [electron/electron#16429]
* Output of `node_modules/.bin/electron --version`: 4.0.1
* Operating System (Platform and Version): OS X El Capitan 10.11.5

**Expected Behavior**
Application should start

**Actual behavior**
Application crashes with:
```
Dyld Error Message:
  Symbol not found: _kSecAttrKeyTypeECSECPrimeRandom
  Referenced from: /Applications/TIDAL.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework
  Expected in: /System/Library/Frameworks/Security.framework/Versions/A/Security
 in /Applications/TIDAL.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework
```

**To Reproduce**
 1. On El Capitan, download Electron 4.0.1 from here: https://github.com/electron/electron/releases/download/v4.0.1/electron-v4.0.1-darwin-x64.zip
 2. Unzip and try to run Electron

**Additional Information**
Related issues: 
 - https://bugs.chromium.org/p/chromium/issues/detail?id=841769
 - https://bitbucket.org/chromiumembedded/cef/issues/2454/cefclient-3440-fails-to-load-on-mac-os-x

**Comment by nornagon** [comment]
Hey @jkleinsc, interesting information in the linked crbug (https://crbug.com/841769). Looks like some versions of the 10.12 SDK have bad symbol availability in the headers. Could you confirm which version we're using?

**Comment by TheHaff** [comment]
+1

**Comment by jkleinsc** [comment]
@nornagon our CI has Xcode 8.3.3, 10.10.3 MacOS SDK, and 10.12.4 MacOS SDK

**Comment by olafuria** [comment]
This is blocking us from updating Tidal app to Electron 4.0

**Comment by torjusb** [comment]
This seems to have been fixed with the 4.0.2 release, so I'll go ahead and close this.
